stats/test/platform-routes.test.js

```diff
@@ -154,24 +119,45 @@ describe('Platform Routes HTTP request handler', () => {
       await assertResponseStatus(res, 200)
       const metrics = await res.json()
       assert.deepStrictEqual(metrics, [
-        { day: '2024-01-11', accepted_measurement_count: '1', total_measurement_count: '2' },
-        { day: '2024-01-12', accepted_measurement_count: '3', total_measurement_count: '4' }
+        { day: '2024-01-11', accepted_measurement_count: 11, total_measurement_count: 13 },
+        { day: '2024-01-12', accepted_measurement_count: 20, total_measurement_count: 22 }
```

Comment:
Same comments as above:

- The relevant test data should be explicitly expressed in the test (not hidden in helpers)
- Please setup the test data so that the request returns the same response data